python/ingestion/cdc_to_bq.py

```diff
@@ -0,0 +1,25 @@
+import json
+import logging
+from pandas import DataFrame
+from .gcs_to_bq_util import load_csv_as_dataframe, append_dataframe_to_bq
+
+
+def write_covid_deaths_to_bq(dataset, table_name, gcs_bucket, filename):
+  """Writes cdc covid death data to BigQuery from bucket
+    
+     dataset: The BigQuery dataset to write to
+     table_name: The name of the biquery table to write to
+     gcs_bucket: The name of the gcs bucket to read the data from
+     filename: The name of the file in the gcs bucket to read from"""
+  try:
+    frame = load_csv_as_dataframe(gcs_bucket, filename)
+
+    # Replace spaces and dashes with underscores in column name and make all characters
+    # in column names lower case.
+    frame.rename(columns=lambda col: col.replace(' ', '_'), inplace=True)
+    frame.rename(columns=lambda col: col.replace('-', '_').lower(), inplace=True)
+    
+    append_dataframe_to_bq(frame, dataset, table_name)
+  except json.JSONDecodeError as err:
```

Comment:
Do you know which of these lines can throw a JSONDecodeError? From reading I don't see any json. It might make sense to get rid of the try/except block completely, or not restrict the exception to JSONDecodeError.

Reply:
I would just get rid of the try/except block. Unrelated to this PR, but we probably want to do a pass through the rest of the ingestion code as well and remove unnecessary try/except blocks that are just catching an error and logging it (eg in [census_to_bq](https://github.com/SatcherInstitute/prototype/blob/master/python/ingestion/census_to_bq.py)). The error itself should typically provide enough context that the extra message isn't necessary, and for rare cases where it's not I'd advocate just rethrowing like this to preserve error flows:
```
except SomeError as e:
  raise RuntimeError("More specific message") from e
```